client/src/components/LoginForm/LoginFormContainer.js

```diff
@@ -22,25 +25,67 @@ const styles = ({
   },
 });
 
+const LoginValidation = yup.object().shape({
+  email: yup
+    .string()
+    .email()
+    .required(),
+  password: yup
+    .string()
+    .min(5)
+    .max(16)
```

Comment:
@irenelee-hll @ivanms1 Do we need a max length for password? Isn't it okay to not have it?